flashfocus/xutil.py

```diff
@@ -74,7 +74,7 @@ def set_opacity(window, opacity, checked=True):
     If opacity is None, request is ignored.
 
     """
-    if opacity:
+    if opacity >= 0:
```

Comment:
I think we probably want `if opacity is not None` instead. `None >= 0` will give a TypeError.